htdocs/core/class/commonobject.class.php

```diff
@@ -7096,18 +7096,17 @@ public function showOptionals($extrafields, $mode = 'view', $params = null, $key
 					$langs->load($extrafields->attributes[$this->table_element]['langfile'][$key]);
 				}
 
-				$colspan = '';
+				$colspan = 1;
```

Comment:
It seems to be a backport.  On new code this is   $colspan = 0;

Reply:
As I said above, there is a PHP warning bug and a display bug. The warning bug has been fixed in develop by the PR I referred to above. But it is not the case for the display bug no : setting $colspan to 0 by default provokes a separator with attribute colspan="1". It therefore doesn't expand to the full width of most of the cards, that have 2 columns.

Call the first part a backport if you want, but the second part is fixed nowhere, needs the first part to be fixed, and anyway, in the first place, the correct way to fix the first part should have been to fix it in v12.0.

Reply:
So my understanding is that there is still a bug into develop (only 1 on 2 were fixed) ?
Can you describe how we can reproduce this second bug on develop ?